tilt-scripts/svm/test_swap.py

```diff
@@ -0,0 +1,95 @@
+import argparse
```

Comment:
i think it would be easier (and better) to start writing the tilt scripts in JS? i think you could probably write this whole script more flexibly (using SDK methods rather than writing endpoints and payloads manually) in JS